### DISCUSSION TITLE
Quest: ID: 11280 - Draconis Gastritis | have a bug

### DISCUSSION THREAD

**Discussion by xp20035** [azerothcore/azerothcore-wotlk#11522] [General]
Hi, 

I'm using your azerothcore server on a linux machine. I noticed that a quest in the game has a bug.
How can I fix this quest? 
The quest is: Draconis Gastritis id:11280

You can accept the quest from the NPC, but when you use the item, the dragon comes and just attacks and doesn't allow you to complete the quest. I've tested it several times.

**Comment by Panda8k** [comment]
Yes, I also encountered this problem

**Comment by Kitzunu** [comment]
Open a bug report. https://github.com/azerothcore/azerothcore-wotlk/issues/new?assignees=&labels=&template=bug_report.yml
